### PR TITLE
Support SLES 15 SP1

### PR DESCRIPTION
**Pull request by innobead**
As titled, need some change to make deployment working on SUSE SLES 15 SP1 to resolve glibc shared libraries compatible on host.

### CODE REVIEW

**Comment by subodh01** [comment]
yaml files look good.
Do you have to install any iscsi or nfs package ?
Could you please include README file in this directory with all these details.

**Comment by innobead** [comment]
> yaml files look good.
> Do you have to install any iscsi or nfs package ?
> Could you please include README file in this directory with all these details.

README.md added. 

Yes, iscsi client package is needed but nfs not because nfs client is already installed in the plugin container image.

File: deploy/Centos/ntnx-csi-node.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       labels:
         app: csi-node-ntnx-plugin
     spec:
-      serviceAccount: csi-node-ntnx-plugin
+      serviceAccountName: csi-node-ntnx-plugin
```
Comment (subodh01) [inline deploy/Centos/ntnx-csi-node.yaml:20]: Why do you have to change serviceAccount to servviceAccountName ?

Comment (innobead) [reply deploy/Centos/ntnx-csi-node.yaml:20]: serviceAccount is deprecated, need to use serviceAccountName instead.

Ref: ds.spec.template.spec

DeprecatedServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.

Comment (subodh01) [reply deploy/Centos/ntnx-csi-node.yaml:20]: Thanks for the info.

**Review by subodh01 — APPROVED** [review]

**Comment by subodh01** [comment]
@innobead we are close to release Nutanix CSI driver v2.0.0. I have created release-2.0 branch for that. It will be great if you can test and updated SLES 15 SP1 deployment yamls for this release. Thanks.

**Comment by innobead** [comment]
> @innobead we are close to release Nutanix CSI driver v2.0.0. I have created release-2.0 branch for that. It will be great if you can test and updated SLES 15 SP1 deployment yamls for this release. Thanks.

Thanks for the effort to make 2.0.0 out :+1:  

Right now I don't have an environment to verify, but I will try if I get one. Thanks.

**Pull request merged**